client/src/reducers/text.js

```diff
@@ -0,0 +1,16 @@
+var TextReducer = (state = {
+	text: null
+}, action) => {
+	switch (action.type) {
+	case 'ADD_TEXT':
+		return {
+			...state,
```

Comment:
This will [only work](http://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties) on Node.js >= 8.6.0. Are we okay with that?